rafthttp/transport.go

```diff
@@ -68,6 +69,11 @@ type Transporter interface {
 	// It is the caller's responsibility to ensure the urls are all valid,
 	// or it panics.
 	UpdatePeer(id types.ID, urls []string)
+	// ActivateTime returns the time that the connection with the peer
+	// of the given id becomes active. If there is no down time since AddPeer,
```

Comment:
If the connection is active since peer was added, it returns the adding time. If the connection is currently inactive, it returns zero time.